src/components/NavigationComponent/NavigationComponent.module.scss

```diff
@@ -0,0 +1,24 @@
+.navigation {
+  height: 100%;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  align-content: center;
+  background: #f6f7f4;
+
+  .navButton {
+    height: 25px;
+    width: 25px;
+    align-self: center;
+    margin: 10px;
+    font-family: "M PLUS Rounded 1c", sans-serif;
+    font-weight: bolder;
+    img {
+      height: 25px;
+      width: 25px;
+    }
+    &:hover {
+      cursor: pointer;
+    }
+  }
+}
```

Comment:
Something is missing from this.  When the app is shrunk down to mobile size (Galaxy s5 or iPhoneX) the right navigation button is off the screen... I'm trying a bunch of different things but nothing is working.  I'll keep digging.

Reply:
Weird..When I chose the Galaxy s5 and iphoneX views, all the icons still show for me..
@rinatoyo @Brianwkhuffman @calmoon808 Would you guys mind taking a look and let me know if this happens on your computers too?

Reply:
yes please check!